### PR TITLE
Upgrade redux-saga from 0.16.2 to 1.1.3

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Upgrades redux-saga and uses upgraded audius-client

### Dragons

There are relatively few native sagas, so any errors i suspect will come from audius-client

### CODE REVIEW

File: src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ type ChallengeRewardsDrawerProps = {
   /** The number of steps the user has to complete in total */
   stepCount?: number
   /** The amount of $AUDIO that is rewarded to the user for completing the challenge */
-  amount: string
+  amount: number
```
Comment (sliptype) [inline src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx:209]: Why did this change?

**Review by sliptype — APPROVED** [review]
Looks good! I was confused for a sec because the pr title says 1.16.2 lol

File: src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx (hunk 1 above)
Comment (dylanjeffers) [reply src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx:209]: yeah i was also curious about that, meant to request marcus' review, cause i think that is around code he is familiar with

Comment (rickyrombo) [reply src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx:209]: Yep, new client change went in that changes this to a number: https://github.com/AudiusProject/audius-client/pull/808

Comment (rickyrombo) [reply src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx:209]: Thanks for changing this for me 😄

Comment (dylanjeffers) [reply src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx:209]: perfect!

**Pull request merged**